processor/deltatorateprocessor/processor.go

```diff
@@ -41,10 +48,57 @@ func (dtrp *deltaToRateProcessor) Start(context.Context, component.Host) error {
 
 // processMetrics implements the ProcessMetricsFunc type.
 func (dtrp *deltaToRateProcessor) processMetrics(_ context.Context, md pdata.Metrics) (pdata.Metrics, error) {
+	resourceMetricsSlice := md.ResourceMetrics()
+
+	for i := 0; i < resourceMetricsSlice.Len(); i++ {
+		rm := resourceMetricsSlice.At(i)
+		ilms := rm.InstrumentationLibraryMetrics()
+		for i := 0; i < ilms.Len(); i++ {
+			ilm := ilms.At(i)
+			metricSlice := ilm.Metrics()
+			for j := 0; j < metricSlice.Len(); j++ {
+				metric := metricSlice.At(j)
+				if _, ok := dtrp.ConfiguredMetrics[metric.Name()]; !ok {
+					continue
+				}
+				if metric.DataType() != pdata.MetricDataTypeSum || metric.Sum().AggregationTemporality() != pdata.AggregationTemporalityDelta {
+					dtrp.logger.Info(fmt.Sprintf("Configured metric for rate calculation %s is not a sum or delta\n", metric.Name()))
```

Comment:
s/is not a sum or delta/is not a delta sum

Reply:
Updated.